cpp/tests/utilities/typed_tests.hpp

```diff
@@ -74,6 +75,19 @@ constexpr auto types_to_ids() {
 using NumericTypes =
     cudf::test::Types<int8_t, int16_t, int32_t, int64_t, float, double>;
 
+/**---------------------------------------------------------------------------*
+ * @brief Provides a list of all timestamp types supported in libcudf for use
+ * in a GTest typed test.
+ *
+ * Example:
+ * ```
+ * // Invokes all typed fixture tests for all timestamp types in libcudf
+ * TYPED_TEST_CAST(MyTypedFixture, cudf::test::TimestampTypes);
```

Comment:
```suggestion
 * TYPED_TEST_CASE(MyTypedFixture, cudf::test::TimestampTypes);
```

Reply:
Fixed this + other instances of this typo.